src/__experimental__/components/numeral-date/numeral-date.stories.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import {
+  array,
+  boolean,
+  withKnobs,
+  text
+} from '@storybook/addon-knobs';
+import { Store } from '@sambego/storybook-state';
+import { action } from '@storybook/addon-actions';
+import NumeralDate from '.';
+import Textbox from '../textbox';
+
+export default {
+  title: 'Test/Numeral Date',
+  component: NumeralDate,
+  decorators: [withKnobs],
+  parameters: {
+    info: {
+      disable: true
+    },
+    knobs: { escapeHTML: false }
+  }
+};
+
+export const Basic = () => {
+  const store = new Store({
```

Comment:
Don't use `@sambego/storybook-state` anymore, all you want to do can be achieved by using hooks in stories now. In your case treat `Basic` like any other component and use hooks